lxd/instance/drivers/driver_lxc.go

```diff
@@ -5673,6 +5673,7 @@ func (d *lxc) Exec(req api.InstanceExecPost, stdin *os.File, stdout *os.File, st
 		return nil, fmt.Errorf("Failed to retrieve PID of executing child process")
 	}
 	d.logger.Debug("Retrieved PID of executing child process", log.Ctx{"attachedPid": attachedPid})
+	d.lifecycle(lifecycle.EXEC, log.Ctx{"attachedPid": attachedPid})
```

Comment:
The PID isn't something that's actually useful in an auditing context. What would be useful though would be the command being run (`req.Command`).